src/content/blog/arm64-apple-silicon-m1-ddev-local-what-does-it-all-mean.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "ARM64! Apple Silicon! M1! DDEV-Local! What does it all mean?"
+title: "ARM64! Apple Silicon! M1! DDEV! What does it all mean?"
 pubDate: 2020-11-18
 summary: A look at Apple’s new chip architecture and what it means for DDEV.
-modifiedDate: 2020-12-16
-modifiedComment: "An Apple Silicon M1 prerelease [is now available](ttps://github.com/ddev/ddev/releases)! See the release notes for installation details."
+modifiedDate: 2023-03-12
+modifiedComment: "DDEV has now been fully native compatible for more than two years, so it seemed time to update this blog post."
```

Comment:
Since this is basically an alert that the post has been updated, it might be better to use this comment to summarize what changed. As it exists now, it basically says “this post is old so it seemed time to update”—which doesn’t describe _how_ it updated and it implies that any old posts should be updated. (Which seems needlessly ambitious for dated content and uncommon among blogs.)

I’d either omit `modifiedComment`, summarize what changed about the article since its 11/18/2020 form, or use this update to point to a _new_ post with an update after 2+ years—which would appear as new content on the homepage, in the blog listing, and for RSS subscribers.

I think this is a slightly more descriptive summary of the change:

```suggestion
modifiedComment: "We’ve updated this post to reflect the continued support for Apple silicon since it was first published."
```